projects/frontend/shared-components/cicd/.gitlab-ci.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# frontend-shared-components-build:
+#   image: "versatiledatakit/vdk-cicd-base-gui:2.0.0"
+#   stage: build
```

Comment:
I'm in favour of building on stage `build`, and releasing on stage `release`, likewise the rest of the vdk root project components